src/SamplesApp/SamplesApp.UITests/UnoSamples_Tests.Keyboard.cs

```diff
@@ -275,6 +275,16 @@ public void TextBox_TextChanged()
 			;
 		}
 
+		[Test]
+		public void TextBox_PageLoadedTest()
+		{
+			Run("UITests.Shared.Windows_UI_Xaml_Controls.TextBoxTests.TextBox_TextChanged");
```

Comment:
did you confirm that without the fix this page crashes?

Reply:
Yes this page crashes without the fix, on iOS